packages/app/src/__tests__/CorePack.test.tsx

```diff
@@ -72,3 +73,54 @@ test('visualize 2D datasets', async () => {
 
   expect(await screen.findByRole('figure', { name: 'twoD' })).toBeVisible();
 });
+
+test('visualize 1D slice of a 3D dataset with and without autoscale', async () => {
+  jest.useFakeTimers('modern');
+  await renderApp();
+  await selectExplorerNode('resilience/slow_slicing');
+
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let fetches succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+
+  userEvent.click(await screen.findByRole('tab', { name: Vis.Line }));
+
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let fetches succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+
+  // Move to other slice to fetch new slice
+  // eslint-disable-next-line prefer-destructuring
+  const d0Slider = screen.getAllByRole('slider', {
+    name: 'Dimension slider',
+  })[0];
+  d0Slider.focus();
+  userEvent.keyboard('{PageUp}');
+  expect(await screen.findByText(/Loading current slice/)).toBeVisible();
+
+  // Let fetches succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+  d0Slider.blur(); // remove focus to avoid state update after unmount
+
+  // Activate autoscale
+  userEvent.click(await screen.findByRole('button', { name: 'Auto-scale' }));
+  expect(await screen.findByText(/Loading entire dataset/)).toBeVisible();
+
+  // Let fetches succeed
+  jest.runAllTimers();
+  expect(await screen.findByRole('figure')).toBeVisible();
+
+  // Check that entire dataset is fetched
+  d0Slider.focus();
+  userEvent.keyboard('{PageUp}');
+  expect(await screen.findByRole('figure')).toBeVisible();
+  d0Slider.blur(); // remove focus to avoid state update after unmount
+
+  jest.runOnlyPendingTimers();
+  jest.useRealTimers();
+});
```

Comment:
New test to test the new auto scale behaviour. It should perhaps be put in `Visualizer.test` ?

Reply:
It's quite intense :sweat_smile: -- I would perhaps split the test in two: "Visualize 3D dataset as line with auto-scale enabled" and  "Visualize 3D dataset as line with auto-scale disabled".

Reply:
Well, in the end, I still need to go through all the previous steps to test auto-scale disabled so I might as well keep the same test.

I changed the comments to be more meaningful instead.